src/windows/cheats.c

```diff
@@ -499,6 +565,81 @@ static void window_cheats_misc_dropdown(rct_window *w, int widgetIndex, int drop
 	game_do_command(0, GAME_COMMAND_FLAG_APPLY, CHEAT_FORCEWEATHER, dropdownIndex, GAME_COMMAND_CHEAT, 0, 0);
 }
 
+enum
+{
+	CONVERT_NUMBER_TO_STRING,
+	CONVERT_STRING_TO_NUMBER,
+};
+
+static void window_cheats_convert_money(int convertMethod, char* text) 
```

Comment:
" They could consume a pointer to whatever string buffer is required." I am not sure what you mean by this, could you clarify? and i'll work on pushing this to "money_to_string" and "string_to_money" like you've mentioned and migrate it to utils.c (or util.c, whichever it's called).

"sscanf for it, as it returns number of bytes/chars consumed." could you also clear this up? if i do your method (about finding a decimal seperator, atoi'ing everything to the left and then right), would i have to make sure everything that's not a number or the decimal seperator gets filtered out beforehand? i would honestly like to make this much simpler because it looks so complex for just a miniscule thing.